web/src/lib.rs

```diff
@@ -1244,21 +1244,21 @@ async fn create_renderer(
     // Try to create a backend, falling through to the next backend on failure.
     // We must recreate the canvas each attempt, as only a single context may be created per canvas
     // with `getContext`.
-    #[cfg(feature = "webgl")]
-    {
-        log::info!("Creating WebGL renderer...");
-        let canvas: HtmlCanvasElement = document
-            .create_element("canvas")
-            .into_js_result()?
-            .dyn_into()
-            .map_err(|_| "Expected HtmlCanvasElement")?;
-        match ruffle_render_webgl::WebGlRenderBackend::new(&canvas, _is_transparent) {
-            Ok(renderer) => {
-                return Ok((builder.with_renderer(renderer), canvas));
-            }
-            Err(error) => log::error!("Error creating WebGL renderer: {}", error),
-        }
-    }
+    // #[cfg(feature = "webgl")]
```

Comment:
Accidentally pushed?
I guess we need a setting to force render backend.

Reply:
We have a setting in the extension, but I'm lazy and always just comment it out in the source and run the demo. :-)

I wonder what the easiest way to handle this for self-hosted would be. I was thinking a TamperMonkey script that injected a Ruffle config into the page. Maybe we could have the Ruffle extension inject its config into the page even if it's otherwise disabled.